.github/workflows/build-irods-centos.yml

```diff
@@ -16,6 +16,13 @@ jobs:
               yum -y update ca-certificates
               yum -y install bind-utils wget epel-release centos-release-scl
               yum -y install gcc g++ libstdc++-static make rpm-build bzip2-devel curl-devel fakeroot openssl-devel pam-devel python-devel unixODBC unixODBC-devel zlib-devel python36-distro
+      - name: Install Flex and Bison
+        run:  |
+              yum -y install centos-release-scl-rh
```

Comment:
This isn't needed since `centos-release-scl` is installed in an earlier step.

Reply:
Updated.